backend/supabase/seed.sql

```diff
@@ -845,6 +848,21 @@ create table if not exists
 
 alter table user_topics enable row level security;
 
+create table if not exists
+  market_ads (
+    id text not null primary key default uuid_generate_v4 (),
+    user_id text not null,
+    market_id text not null,
+    foreign key (market_id) references contracts (id),
+    funds numeric not null,
+    cost_per_view numeric not null,
+    created_at timestamp not null default now(),
+    embedding vector (1536) not null
+  );
+
+alter table market_ads
+add constraint market_ads_unique_market_id unique (market_id);
```

Comment:
I think this line will fail when run again. You should add the contraint to the bottom of the table definition:

create table if not exists
  market_ads (
    id text not null primary key default uuid_generate_v4 (),
    user_id text not null,
    market_id text not null,
    foreign key (market_id) references contracts (id),
    funds numeric not null,
    cost_per_view numeric not null,
    created_at timestamp not null default now(),
    embedding vector (1536) not null,
    unique (market_id)
  );